composer.json

```diff
@@ -20,9 +20,9 @@
         }
     ],
     "require": {
-        "php": "~5.6|~7.0",
+        "php": ">=7.1",
```

Comment:
Is this change required to make it work with Laravel 6? Maybe leave it as `~5.6|~7.0` if that will let it work for users on older php versions.